### ISSUE TITLE
(Feature Suggestion) Capability to Edit Vertex Position of Saved User Defined Geometry.

### ISSUE THREAD

**Issue by esacci** [CCI-Tools/cate#586]
Feature suggestion - capability to edit points of existing user define geometries. If a user has spent a lot of effort creating a user-defined polygon, then they may wish to edit the location of the vertices of that (saved) polygon at a later time. At the moment, it is not obvious how to do this ; perhaps this functionality does not exist.

**Comment by forman** [comment]
No, it does not exist yet. Users currently must delete a selected geometry then draw a new one (see #401).

A workaround is to open `~/.cate/preferences.json` in a text editor, then scroll to the element `"placemarkCollection"`, then find the [GeoJSON](http://geojson.org/) *feature* whose geometry you want to change using search term `"title": "<my-title>"` where `<my-title>` is the name of the place displayed in the GUI. In its `"geometry"` element you can edit the feature's coordinates.
